client/src/components/Login.jsx

```diff
@@ -50,19 +51,22 @@ const Login = ({ showLogin, setShowLogin }) => {
   };
 
   return (
-    <div className='loginForm'>
-      <h1>Login</h1>
-      <form onSubmit={(e) => handleSubmit(e)}>
-        <input type='text' name='username' placeholder='Username' />
-        <br></br>
-        <input type='text' name='password' placeholder='Password' />
-        <br></br>
-        <button type='submit'>Login</button>
-        <button className='btn' onClick={() => changePage('main')}>
-          Main
-        </button>
-        <button onClick={() => changePage('signup')}>Sign up!</button>
-      </form>
+    <div className='background'>
+      {/* <img src={leaf} height='50px' width='50px'></img> */}
+      <div className='loginForm'>
```

Comment:
If this is a "loginForm" should it be a form element? Or should this be renamed to something like loginContainer or loginCard?